Client/Components/SingleAnswerComponent.razor

```diff
@@ -1,133 +1,131 @@
 @inject IJSRuntime js
-<div class="row">
-    <div class="column" style="width:5%">
-        <div >
-            <div class="answer-rating"> 
-                <div>
-                    @if (Answer.Mark == 1)
-                    {
-                        <button class="triangle-up" style="color:orange" disabled="disabled" />
-                    }
-                    else
-                    {
-                        <button class="triangle-up" @onclick="@(async() => await IncreaseRating())" disabled="@isNonInteractive" />
-                    }
-                </div>
-                <div>
-                    @Answer.Rating
-                </div>
-                <div>
-                    @if (Answer.Mark == -1)
-                    {
-                        <button class="triangle-down" style="color:orange" disabled="disabled" />
-                    }
-                    else
-                    {
-                        <button class="triangle-down" @onclick="@(async() => await DecreaseRating())" disabled="@isNonInteractive" />
-                    }
-                </div>
-            </div>
-            <div style="flex-direction:row">
-                <div>
-                    @Answer.UserResponseDTO.Nickname
-                </div>
-                @if (Answer.IsModified)
-            {
-                <div>
-                    Modifed
-                </div>
-            }
-                
-            </div>
-        </div>
-    </div>
+@inject NavigationManager navigationManager
 
 
-    @if (EditMode)  
+<div class="header-info">
+    <div class="publish-date">
+        @Answer.PublishDate.ToShortDateString()
+    </div>
+    <div>
+        answered by
+    </div>
+    <div>
+        <a href="/users/@Answer.User.IdUser">
+            <img class="user-avatar" src="@Answer.User.Avatar" alt="Avatar" />
+        </a> 
+    </div>
+    <div>
+        <div class="user-nickname">
+            @Answer.User.Nickname
+        </div>
+        <div>
+            @Answer.User.Points points
+        </div>
+    </div>
+</div>
+<div class="grid-container">
+    @if (ActiveUser == null)
     {
-        <AnswerTextComponent Text="@Answer.Text" Publish="UpdateAnswer" Discard="SetEditMode"></AnswerTextComponent>
+        <div class="row-line">
+        </div>
+        <div class="row-unloggedUser">
+            <UnloggedUserComponent></UnloggedUserComponent>
+        </div>
     }
-    else
-    {
-        <div class="answer-text">
-            @if (Answer.IdAnswer == ActiveResponse)
+    <div class="row-answer">
+        <div class="text @(ActiveUser==null?"blur":"")">
+            @Answer.Text
+        </div>
+
+        <div class="footer @(ActiveUser==null?"blur":"")">
+            <button class="quanda-icon icon-arrowup answer-rating-arrow" style="@(Answer.IsLiked==1?"color:#FFAD33":"")" @onclick="@(async() => await IncreaseRating())" disabled="@(isNonInteractive || OwnerOfAnswer || Answer.IsLiked == 1)" />
```

Comment:
Please use css classes instead of styles as it will be easier to change them later.